language/Portuguese_Brazil/strings.po

```diff
@@ -1680,8 +1689,8 @@ msgid ""
 "Roms Audit Pass\tDisabled\n"
 "Samples Audit Pass\tDisabled\n"
 msgstr ""
-"Auditoria de Roms\tDesligada\n"
-"Auditoria de Amostras\tDesligada\n"
+"Auditoria de Roms Passou\tDesabilitada\n"
+"Auditoria de Amostras Passou\tDesabilitada\n"
```

Comment:
Are you sure the literal translation is actually better here?  "Desligada" seems to convey the sense of what it's saying.

Reply:
I think both are good here. But I'd prefer @katananja 's "Disabled"/"Desabilitada".

What really concerns me here is that "Passou" that was added. It means "Passed". I'm not sure that's the intended meaning.

@cuavas can you clarify where and how this string shows up in the UI ?

Reply:
It's part of the info box in the system selection UI - you can configure it to automatically audit ROMs/sound samples.  Depending on whether you have it enabled or disabled and whether it passes or fails, it will show one of:
* Roms Audit Pass - OK
* Roms Audit Pass - BAD
* Roms Audit Pass - Disabled

And likewise for samples.

The English is already poorly worded.  The word "pass" refers to passing over the ROMs/samples when auditing, not that the audit passed.  Also, adding "Passou" to the translation for the case where the automatic audit is disabled without also adding it for the OK/BAD cases doesn't make sense to me.